packages/react/src/components/Calendar/CalendarMonth/CalendarMonth.base.tsx

```diff
@@ -218,6 +225,7 @@ export const CalendarMonthBase: React.FunctionComponent<ICalendarMonthProps> = p
           aria-atomic={true}
           // if this component rerenders when text changes, aria-live will not be announced, so make key consistent
           aria-live="polite"
+          key={yearString}
```

Comment:
Is the key change related to the JAWS behavior somehow?

Reply:
@jhellesen Yes; I'm not entirely sure why, but without the key, if we used the arrow navigation buttons, JAWS would read the header button's updated aria label (the button/div that shows what month/year/etc. it is) before the navigation button's aria label. By adding the key, JAWS now reads it after the navigation button and when the user isn't doing anything else, aka "politely". The key was already present in the CalendarDay component, so I'm adding it to CalendarMonth and CalendarYear here. 

I'll update the PR description to be a little more clear about this!